department-of-veterans-affairs/caseflow
# Khalin/appeals 41775 eFolder Upload Failed Banner

Resolves [Jira Issue Title](https://jira.devops.va.gov/browse/APPEALS-41775)

# Description
As a Mail Team user, Superuser, and Supervisor who uses Caseflow, when I return to a previous Correspondence Intake that failed to upload to eFolder, the banner stating the previous eFolder Upload failed still persists when you return to the page, so that I am aware the action failed during the last intake. 

## Acceptance Criteria

## Testing Plan

1. Login as MAIL_TEAM_SUPERVISOR_MAIL_INTAKE_USER
2. Click on "Switch Views"
3. Click on "Correspondence Cases"
4. Click on a Correspondence case that has a package document type of 10182 (first 10 cases)
5. Scroll to the bottom and click on the "Create record" button
6. Click on the "Continue" button
7. Scroll to the bottom and click on the "Continue" button
8. Scroll to the bottom and click on the "Submit" button
9. Click on the "Confirm" button
10. Check to see the "failed to upload to the eFolder" banner at the top
11. Copy the current URL
12. Scroll to the bottom and click on the "Cancel" at the bottom left
13. When the next page loads paste the copied URL back where it was and navigate to that page
14. Check to see if the "failed to upload to the eFolder" banner is STILL at the top

# Frontend
## User Facing Changes
**The banner stating the previous eFolder Upload failed still persists when you return to the page.**
<img width="1378" alt="Screenshot 2024-03-20 at 9 29 08 AM" src="https://github.com/department-of-veterans-affairs/caseflow/assets/134650238/df2bc08a-a9c0-4b0f-b74d-f61d64d24ce7">
